### ISSUE TITLE
omnisharp 1.34.4 is not downloading and installing omnisharp.exe to file path getting following error

### ISSUE THREAD

**Issue by abdulkareemmoodys** [dotnet/vscode-csharp#3327]
after installing c# extension i am getting following error and not able to proceed with code debugging. I found out that OmniSharp.exe is not present in the path its referring to. can any one help me how to get OmniSharp.exe may be the error is because this file is not copied automatically while installing the C# extension. Thanks in advance for help.

"Starting OmniSharp server at 10/7/2019, 8:28:36 PM
Target: d:\Learning\pluralsight\C#_Fundamentals\Lab\GardeKBook

OmniSharp server started.
Path: C:\Users\kareabd.vscode\extensions\ms-vscode.csharp-1.21.4.omnisharp\1.34.4\OmniSharp.exe
PID: 25720

'C:\Users\kareabd.vscode\extensions\ms-vscode.csharp-1.21.4.omnisharp\1.34.4\OmniSharp.exe' is not recognized as an internal or external command,
operable program or batch file."

**Comment by rchande** [comment]
@abdulkareemmoodys Can you manually downlaod https://omnisharpdownload.blob.core.windows.net/ext/omnisharp-win-x64-1.34.4.zip and unzip it to the directory mentioned in the error message (`C:\Users\kareabd.vscode\extensions\ms-vscode.csharp-1.21.4.omnisharp\1.34.4\`)?

**Comment by cartermp** [comment]
@abdulkareemmoodys Does this still reproduce for you with the latest version of the plugin?

**Comment by willie001** [comment]
Same problem here but with later version: c:\Users\ez002116\.vscode\extensions\ms-dotnettools.csharp-1.22.1\.omnisharp\**1.35.3**\OmniSharp.exe

**Comment by mcm-ham** [comment]
I'm running into this issue on Windows 10 Arm. The `C:\Users\User\.vscode\extensions\ms-dotnettools.csharp-1.23.11\.omnisharp\1.37.8` folder was missing. Downloading omnisharp from https://github.com/OmniSharp/omnisharp-roslyn/releases and placing into that folder worked. Note needed to run `dir -Path 1.37.8 -Recurse | Unblock-File` in powershell to unblock all the dll files before it worked.

**Comment by JoeRobich** [comment]
A lot has changed with packaging and acquisition of required component since this issue was opened. If you are still running into problems, please open a new issue.
